programs/zstd.1.md

```diff
@@ -191,6 +191,9 @@ the last one takes effect.
     This is the default behavior.
 * `-r`:
     operate recursively on directories
+* `--filelist=FILE`
```

Comment:
nit: `--file-list` is more consistent with the rest of the CLI args.

Reply:
not sure why, but when I write it as` --file-list=`, it does not feel right ...